examples/policies/getting-started/dns-matchname.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "fqdn"
+specs:
+  - endpointSelector:
+      matchLabels:
+        org: empire
+        class: mediabot
+    egress:
+    - toFQDNs:
+        - matchName: "api.twitter.com"  
+  - endpointSelector:
+      matchLabels: {}
```

Comment:
This is super dangerous as you are putting all pods in that namespace into default deny at egress. Somebody running this demo in a namespace with other deployments will result in breakage of all those pods. Match on mediabot.

Reply:
done